src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -2570,6 +2570,26 @@ class C
                 expectedIndentation: 8);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        public async Task DontCreateIndentOperationForBrokenBracketedArgumentList()
+        {
+            var code = @"
+class Program
+{
+    static void M()
+    {
+            string (userInput == ""Y"")
```

Comment:
I figured about the parsing, it's just a bit surprising, that's all.

Then I would move the string back four spaces, just so people don't think it's part of the repro in the future.